cfme/utils/ftp.py

```diff
@@ -671,3 +671,13 @@ def download(self, target=None):
             target path
         """
         return self.client.download(self.name, target)
+
+    @property
+    def filesize(self):
+        """Return file size"""
+        # Switch to Binary mode
+        self.client.ftp.sendcmd("TYPE i")
+        size = self.client.ftp.size(self.path)
```

Comment:
I think this should be wrapped in a try:except:finally, so that the ASCII switch happens if/when the size call fails for some reason.